modules/logging-release-notes-5-8-16.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-16_{context}"]
+= Logging 5.8.16
+This release includes link:https://access.redhat.com/errata/RHBA-2024:10989[RHBA-2024:10989].
```

Comment:
Our 5.8 release consists of a RHEL8 and a RHEL9 part. The release notes are the sum of both advisories. See #85216 for an example.

The linked advisory is the RHEL8 variant. The RHEL9 one is this: https://errata.devel.redhat.com/advisory/143685

Reply:
Ok cool, thank you, and actually upon double checking the errata, I realize that I needed to add https://access.redhat.com/security/cve/CVE-2024-9287 and https://access.redhat.com/security/cve/CVE-2024-11168 for the RHEL8 part.